src/styles/blocks/card.css

```diff
@@ -0,0 +1,122 @@
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  margin: 20px auto;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  margin: 16px 3px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  margin-top: 40px;
+
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+
+  color: var(--color-dark);
+}
+
+.card__code {
+  margin-top: 4px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-gray);
+}
+
+/* #region Review */
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-top: 16px;
+}
+
+.card__review-text {
+  display: flex;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+
+  color: #000;
+}
+/* #endregion */
+
+/* #region Price */
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 24px;
+}
+
+.card__price-title {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+
+  color: var(--color-gray);
+}
+
+.card__price-price {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  line-height: 18px;
+
+  color: var(--color-dark);
+}
+/* #endregion */
+
+/* #region Button */
+.card__button {
+  margin-top: 16px;
+  background: var(--color-blue);
+  border: 1px solid rgba(0,0,0,0);
+  border-radius: 5px;
+}
+
+.card__button-text {
+  display: flex;
+  justify-content: center;
+
+  font-family: Roboto , sans-serif;
+  text-decoration: none;
+  line-height: 38px;
```

Comment:
no!
Please center the text using align-items

Reply:
I used line-height so the content can dictate the height of the element not for center it .

Reply:
@IoghiStudio with flex you could define
```
height: 38px;
align-items: center;
justify-content: center;
```
and everything is centered properly
let's do this